### DISCUSSION TITLE
级联聚焦时，使用受控组件的方式，无法控制浮层（popupVisible）显示出来

### DISCUSSION THREAD

**Discussion by tangtang09** [alibaba/formily#1777] [Q&A 2.x] [answered]
[https://codesandbox.io/s/bitter-wood-25svi?file=/App.tsx](url)

**Comment by janryWang** [comment, marked as answer]
文档都没看过吗？不支持受控组件模式

https://v2.formilyjs.org/zh-CN/guide/advanced/controlled
